code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -3951,6 +3951,25 @@
 	required_reagents = list(PICCOLYN = 1, INACUSIATE = 1, SUGAR = 1)
 	result_amount = 3
 
+/datum/chemical_reaction/bumcivilian
+	name = "Bumcivilian"
+	id = BUMCIVILIAN
+	result = BUMCIVILIAN
+	required_reagents = list(IRON = 1, SACIDS = 1) //..5.05 Mg
+	result_amount = 1
+
+/datum/chemical_reaction/bumcivilian/required_condition_check(datum/reagents/holder)
+	for(var/obj/item/device/deskbell/B in view(3,get_turf(holder.my_atom)))
+		if(world.time - B.last_ring_time <= 30)
+			return 1
+
+/datum/chemical_reaction/bumcivilian/on_reaction(var/datum/reagents/holder, var/created_volume)
+	..()
+	var/datum/reagent/bumcivilian/B = locate(/datum/reagent/bumcivilian) in holder.reagent_list
+	B.mute_time = world.time
+	for(var/mob/M in range(5,get_turf(holder.my_atom)))
+		M.Mute(B.mute_duration/30) //ticks to seconds i THINK
```

Comment:
life ticks once every 2 seconds, so for example a value of 5 mute would be 10 seconds - I have no idea what your conversion is aiming for; 300 / 30 = 10 so this would be 10 ticks = 20 seconds -- all very roundabout though; since you don't operate on mute duration at all, you could just make mute_duration 10 to begin with instead of doing a division operation here

I should mention that this is weirdly inconsistent. The mute zone lingers in one place -- so things that arrive late don't make sound and things that leave the area DO make sound. But the SPEECH muting is the opposite. Leave the zone? Still can't talk; move into area late, CAN talk. This feels very bad.

Reply:
Did this in a much more consistent manner in the latest commit, now using the same timer as the item muting effect.